frontend/simple/views/components/Voting.vue

```diff
@@ -178,7 +178,10 @@ import { toPercent } from '../utils/filters'
 export default {
   name: 'Voting',
   props: {
-    proposal: Object
+    proposal: Object,
+    onVoteAgainst: Function,
+    onVotedFor: Function,
```

Comment:
is it intentional that we have _vote_ when against and _vote**D**_ when for?

Reply:
Ups no! Fixed!